frontend/src/metabase/query_builder/containers/QueryBuilder.unit.spec.tsx

```diff
@@ -44,13 +48,45 @@ const TEST_CARD = createMockCard({
   },
 });
 
+const TEST_MODEL_CARD = createMockCard({
+  dataset_query: {
+    database: SAMPLE_DB_ID,
+    type: "query",
+    query: {
+      "source-table": ORDERS_ID,
+      limit: 1,
+    },
+  },
+  dataset: true,
+  display: "scalar",
+});
+
 const TEST_NATIVE_CARD = createMockCard({
   dataset_query: createMockNativeDatasetQuery({
     database: SAMPLE_DB_ID,
   }),
 });
 
-const TEST_DATASET = createMockDataset();
+const TEST_DATASET: Dataset = createMockDataset();
```

Comment:
what if we specify this type right in the `createMockDataset`?

Reply:
yep, good call 😄